scania-digital-design-system/tegel
# docs(installation): updated installation/usage guides

**Describe pull-request**  
This PR updates our installation guides in Storybook. After this is merged we will have one story per installation guide:
 - Javascript
 - Angular
 - React

**Solving issue**  
Fixes: [CDEP-2771](https://tegel.atlassian.net/browse/CDEP-2771)

**How to test**  
1. Go to Storybook
2. Check in Installation
